fhir-server/liberty-config/configDropins/disabled/datasource-proxy.xml

```diff
@@ -0,0 +1,13 @@
+<server>
+    <library id="fhirSharedLib">
+        <fileset dir="${shared.resource.dir}/lib/derby" includes="*.jar"/>
+        <fileset dir="${shared.resource.dir}/lib/db2" includes="*.jar"/>
+        <fileset dir="${shared.resource.dir}/lib/postgresql" includes="*.jar"/>
+        <!-- Copy the fhir-persisence-proxy jar and its dependencies to this dir to enable it -->
+        <fileset dir="${shared.resource.dir}/lib/fhir" includes="*.jar"/>
+    </library>
+    
+    <dataSource id="fhirProxyDataSource" jndiName="jdbc/fhirProxyDataSource" type="javax.sql.XADataSource" validationTimeout="30s">
+        <jdbcDriver javax.sql.XADataSource="com.ibm.fhir.persistence.proxy.FHIRProxyXADataSource" libraryRef="fhirSharedLib"/>
+    </dataSource>
+</server>
```

Comment:
team decision:  remove this.  also remove fhri-persistence-proxy from fhir-parent.

Reply:
the problem is that fhir-core and fhir-config are in our webapp now, so including them here could cause classloader issues

Reply:
I manually confirmed that this is working just fine as a dropinConfig/override.  I think it works because the FHIRProxyXADataSource isn't even available to the webapp's classpath.  Its handled just like any other db driver.  Our webapp sticks to the javax.sql interface and I think we avoid the classloader issues.

Reply:
Scratch that, it was working for the default tenant (default datastore), but its failing for other tenants in a very interesting way.  Because the db driver's FHIRRequestContext is independent of the webapp's FHIRRequestContext, calls from the FHIRProxyXADataSource end up instantiating a brand new FHIRRequestContext and therefor it doesn't have the thread-local tenantid or datasource id variables set.  I *might* be able to fix it by removing our use of the FHIRRequestContext from FHIRProxyXADataSource, but its probably more pain than its worth.

Reply:
OK, just pushed a change to more completely remove fhir-persistence-proxy.  I chose not to remove it from fhir-parent at this time, but we will no longer package it in the shared resources directory nor will we provide guidance on how to enable it.